framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -470,3 +492,16 @@ def add_request(self, request: bytes):
         """
         self.logger.debug("Received request: {}".format(request))
         self.request_queue.put_nowait(request.decode())
+
+
+async def get_system_nodes():
+    try:
+        lc = local_client.LocalClient()
+        try:
+            result = await control.get_nodes(lc)
+        finally:
+            lc.transport.close()
+
+        return [node['name'] for node in result['items']]
+    except Exception:
+        raise WazuhInternalError(3012)
```

Comment:
I would move this function to `control.py`

Reply:
Done